packages/backend/src/open_payments/invoice/model.ts

```diff
@@ -25,4 +26,14 @@ export class Invoice extends BaseModel {
   public description?: string
   public expiresAt!: Date
   public readonly amount!: bigint
+  // The "| null" is necessary so that `$beforeUpdate` can modify a patch to remove the webhookId. If `$beforeUpdate` set `webhookId = undefined`, the patch would ignore the modification.
+  public webhookId?: string | null
+  public webhookAttempts!: number
+
+  $beforeUpdate(opts: ModelOptions, queryContext: QueryContext): void {
+    super.$beforeUpdate(opts, queryContext)
+    if (opts.old && opts.old['active'] === true && this.active === false) {
+      this.webhookId = uuid()
```

Comment:
Why reassign the `webhookId` instead of just assigning it once (or using the invoice id)?